src/screens/GuidesAndTips/GuidesAndTips.js

```diff
@@ -0,0 +1,287 @@
+  },
+  {
+    type: 'photo',
+    headline: 'לחץ אוויר משפר את נסיעתכם',
+    text: 'איך לחץ אוויר משפיע על הנסיעה שלכם? לחץ אוויר הוא אחד הגורמים שהכי משפיעים על נסיעה בטוחה כחיג לאורח  דלחכ מהחשי אחרוםפ חעל מממש חיהמ לאוכח  לחכוי מחרמ לכישחאחיעל יאנשיל חאוטרמ צשלחרת לחעילרי חגואיחדמ.',
+    thumbnail: require('./components/images/TirePressure.jpg'),
+    detailedPhoto: require('./components/images/Punc.jpg'),
+    detailHeadline: 'התמודדות עם תקר',
+    detailText: 'הועניב היושבב שערש שמחויט - שלושע ותלברו חשלו שעותלשך וחאית נובש ערששף. זותה מנק הבקיץ אפאח דלאמת יבש, כאנה ניצאחו נמרגי שהכים תוק,הועניב היושבב שערש שמחויוחאית נובש ערששף. זותה מנק הבקיץ אפאח דלאמתרא התידם הכייר וק.'
+  },
+  {
+    type: 'video',
+    headline: 'ביטוח לרכב',
+    text: 'האם אתם באמת יודעים הכל על ביטוחי רכב? טיפים שכל נהג או נהגת צריכים לדעת כדי להיות צרכנים חכמים בכל מה שנוגע לביטוחי רכב. הועניב היושבב שערש שמחויט - שלושע ותלברו חשלו שעותלשך וחאית נובש ערששף. זותה מנק הבקיץ אפאח דלאמת יבש, כאנה ניצאחו נמרגי שהכים תוק, הדש שנרא התידם הכייר וק.',
+    thumbnail: require('./components/images/Insurance.png'),
+    subHeadline: 'איך לבחור ביטוח?', 
+    anotherText: 'הועניב היושבב שערש שמחויט - שלושע ותלברו חשלו שעותלשך וחאית נובש ערששף. זותה מנק הבקיץ אפאח דלאמת יבש, כאנה ניצאחו נמרגי שהכים תוק, הדש שנרא התידם הכייר וק.'
+  },
+];
+
+const Card = ({ item, onExpand }) => {
+  return (
+    <TouchableOpacity style={styles.card} onPress={() => onExpand(item)}>
+      <Image source={item.thumbnail} style={styles.cardImage} />
+      <Text style={styles.cardTitle}>{item.headline}</Text>
+      <Text style={styles.cardText} numberOfLines={2} ellipsizeMode='tail'>{item.text}</Text>
+    </TouchableOpacity>
+  );
+};
+
+const GuidesAndTips = () => {
+  const [expanded, setExpanded] = useState(false);
+  const [currentCard, setCurrentCard] = useState(null);
+  const [isPlaying, setIsPlaying] = useState(false); // Tracks the playing state
+
+  const handleExpand = (card) => {
+    setExpanded(true);
+    setCurrentCard(card);
+    setIsPlaying(false); // Stop the video when changing cards
+  };
+
+  const handleBackPress = () => {
+    setExpanded(false);
+    setCurrentCard(null);
+    setIsPlaying(false); // Ensure video stops when closing the modal
+  };
+
+  const togglePlayback = () => {
+    setIsPlaying(!isPlaying);
+  };
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.header}>
+        <Text style={styles.headerText}>מדריכים וטיפים</Text>
+        <TouchableOpacity style={styles.closeButton} onPress={handleBackPress}>
+          <Image style={styles.closeIcon} source={require('./components/images/exit.png')} />
+        </TouchableOpacity>
+      </View>
+      <Text style={styles.guidesTitle}>מדריכים</Text>
+      <ScrollView>
+        {data.map((item, index) => (
+          <Card key={index} item={item} onExpand={handleExpand} />
+        ))}
+      </ScrollView>
+      {expanded && (
```

Comment:
{expanded &&

This code is unnecessary; the model itself knows whether to be displayed or not with visible={expanded}.